show/main.py

```diff
@@ -1119,7 +1122,7 @@ def users(verbose):
 @click.option('--redirect-stderr', '-r', is_flag=True, help="Redirect an intermediate errors to STDERR")
 def techsupport(since, global_timeout, cmd_timeout, verbose, allow_process_stop, silent, debug_dump, redirect_stderr):
     """Gather information for troubleshooting"""
-    cmd = "sudo timeout -s SIGTERM --foreground {}m".format(global_timeout)
+    cmd = "sudo timeout --kill-after=5m -s SIGTERM --foreground {}m".format(global_timeout)
```

Comment:
as there is a constant of 300 seconds, can we use it here as well?

Reply:
Changed